### DISCUSSION TITLE
疑惑gocqhttp配置文件中的ProxyRewrite如何填写

### DISCUSSION THREAD

**Discussion by shitianshiwa** [Mrs4s/go-cqhttp#780] [Q&A]
https://github.com/Mrs4s/go-cqhttp/blob/master/global/config.go#L168
不清楚是否有作用，还是只是无效果设置？

**Comment by YumeMichi** [comment]
看了一下代码 好像确实就一两个函数有用到？其他绝大部分http请求应该都在miraigo那边

**Comment by Mrs4s** [comment]
这个会让应用层的http请求走代理, 比如`[CQ:image]` 的 `url`.

**Comment by shitianshiwa** [comment]
尝试过以下填写方法，均无效
proxy_rewrite:http://127.0.0.1:1080
proxy_rewrite:127.0.0.1:1080
proxy_rewrite:"http://127.0.0.1:1080"
proxy_rewrite:"127.0.0.1:1080"
